executor/runtime/types/pod.go

```diff
@@ -587,6 +588,10 @@ func (c *PodContainer) ServiceMeshEnabled() bool {
 	return c.serviceMeshEnabled
 }
 
+func (c *PodContainer) TrafficSteeringEnabled() bool {
+	return c.trafficSteeringEnabled
```

Comment:
This is missing something. It is never set!

There needs to be some code somewhere in pod.go that looks for a special Annotation and sets this to be true.

This is the "new" style of attributes with pods (aka v1 pod schema)